config/schemas/form_response.json

```diff
@@ -14,16 +13,6 @@
   ],
   "additionalProperties": false,
   "properties": {
-    "medical_equipment": {
-      "$ref": "#/definitions/yes_no"
-    },
-    "medical_equipment_type": {
-      "type": "string",
-      "enum": [
-        "Personal protective equipment (PPE)",
-        "Coronavirus tests or testing facilities"
-      ]
-    },
```

Comment:
Again, as part of the A-AB-B deployment, should these not be deleted until everyone who already has these in the session submits their answers, so we don't trigger a schema error when submitting?

Reply:
Good call - thanks @steveholmes 👍